x-pack/packages/kbn-elastic-assistant/impl/knowledge_base/knowledge_base_settings_management/use_knowledge_base_table.tsx

```diff
@@ -19,6 +19,56 @@ import * as i18n from './translations';
 import { BadgesColumn } from '../../assistant/common/components/assistant_settings_management/badges';
 import { useInlineActions } from '../../assistant/common/components/assistant_settings_management/inline_actions';
 import { isSystemEntry } from './helpers';
+import { useAsync } from 'react-use';
+
+const AuthorColumn = ({entry}:{entry: KnowledgeBaseEntryResponse}) => {
```

Comment:
There are a bunch of linting errors in this file that I'm surprised your linter didn't catch. I wonder if we lint on type check